web/client/src/golden-layout/golden-layout-adapter.vue

```diff
@@ -105,26 +117,31 @@ const loadGLLayout = async (layoutConfig: LayoutConfig) => {
         | RowOrColumnItemConfig[]
         | StackItemConfig[]
         | ComponentItemConfig[];
+
         for (const itemConfig of content) {
```

Comment:
Coud try to use ```flatMap```. It make the code mor readable.

Reply:
Imo it only made it slightly better, but okay

Reply:
I have resorted to using forEach (function), as:
1. I do not want to add anything to the contents if the itemConfig type is Component (so map is not that fitting here) and
2. Because of 1. trying to force this into a flatMap resulted in hard to debug infinite loops which made the dev-server crash

All in all i have gone from `for (const itemConfig of content` to `content.forEach(...)`. I also added a bit of typing, which made the code more readable in itself.